substrate/client/db/src/bench.rs

```diff
@@ -45,16 +42,16 @@ use std::{
 	sync::Arc,
 };
 
-type State<B> = DbState<B>;
+type State<H> = DbState<H>;
 
-struct StorageDb<Block: BlockT> {
+struct StorageDb<Hasher: Hash> {
```

Comment:
```suggestion
struct StorageDb<Hasher> {
```

Never bound a struct definition generic type if not required.